### PR TITLE
Complete test tasks

### PR DESCRIPTION
**Pull request by greg-grigorjevs**
Dear Claviz,

I am happy to submit my solutions to the test tasks.

Grigorijs Grigorjevs

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,60 @@
...
+const { getXlsxStream } = require("xlstream");
 
-app.get('/', function (req, res) {
-    res.send('Hello, World!');
+app.get("/", function (req, res) {
+    res.send("Hello World");
 });
 
+app.get("/btc", async (req, res) => {
+    await request(
```
Comment (jansivans) [inline index.js]: Why `await` is used here?

File: index.js [hunk 2]
```diff
@@ -1,8 +1,60 @@
...
+
+    const capital = countries[i].city;
+    res.send(`The capital of ${country} is ${capital}`);
+});
+
+app.post("/excel-sum", upload.any(), async (req, res) => {
+    const file = req.files[0];
+    let sum = 0;
+
+    await (async () => {
```
Comment (jansivans) [inline index.js]: Why such construction is used here?

File: index.js [hunk 3]
```diff
@@ -1,8 +1,60 @@
...
+        const stream = await getXlsxStream({
+            filePath: `./${file.path}`,
+            sheet: 0,
+        });
+
+        sum = await new Promise((resolve, reject) => {
```
Comment (jansivans) [inline index.js]: Why are using `Promise` here?

**Review by jansivans — COMMENTED** [review]
Hi! 
Thank you for your solution.
It seems to work, but I have got several questions about your code.
Mainly about strange `async/await` usage.
Can you please answer them and improve your solution if you can?
Thank you in advance.

**Comment by greg-grigorjevs** [comment]
Hi. Thank you for your review. I removed the unnecessary async/await structures and used async iterator instead of Promise in the last exercise. I hope it made my solution better.

**Comment by jansivans** [comment]
Now it is perfect, thanks.

**Pull request closed without merging**